packages/app-desktop/gui/NoteList/utils/useRefocusOnDeletion.ts

```
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import usePrevious from '@joplin/lib/hooks/usePrevious';
+
+const useRefocusOnDeletion = (
+	noteCount: number,
+	selectedNoteIds: string[],
+	focusedField: string,
+	focusNote: (noteId: string)=> void,
```

ESLint

[error] 'noteId' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)